packages/server/modules/auth/tests/apps.spec.js

```diff
@@ -19,6 +19,7 @@ const {
 const { Scopes } = require('@/modules/core/helpers/mainConstants')
 const { updateDefaultApp } = require('@/modules/auth/defaultApps')
 const knex = require('@/db/knex')
+const cryptoRandomString = require('crypto-random-string')
```

Comment:
Good to see more of these being cleaned up 👍 There's a lot of cross test pollution in `server` 🥲